core/scripts/runDockerVoting.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# Must run from the core/ directory.
+node ./scripts/InitDockerConfig.js
+while true; do $(npm bin)/truffle exec ./scripts/Voting.js "$@"; sleep 60; done
```

Comment:
Two tweaks in this loop:
 * The script immediately executes
 * The output goes to `stdout`